src/main/java/com/fundynamic/d2tm/Game.java

```diff
@@ -35,6 +35,8 @@ public void initStatesList(GameContainer container) throws SlickException {
                 TILE_HEIGHT
         );
 
+        container.setVSync(true);
```

Comment:
It does give me tearing effects on high framerates :(

On Sat, Feb 21, 2015 at 12:07 AM, Arjen van der Ende
<notifications@github.com> wrote:

>> @@ -35,6 +35,8 @@ public void initStatesList(GameContainer container) throws SlickException {
>>                  TILE_HEIGHT
>>          );
>>  
>> +        container.setVSync(true);
> I know we spoke about this, but I removed this on purpose yesterday. Since the lagginess is fixed on your computer I think it's nice to allow a high framerate.
> On the other hand, we'll probably make it configurable later on :wink: 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/stefanhendriks/dune2themaker4j/pull/42/files#r25111209